### ISSUE TITLE
Space handling of VarOp property in rules might lead to false positives.

### ISSUE THREAD

**Issue by Angus71** [priv-kweihmann/oelint-adv#428]
While fixing  a problem in the `rule_vars_override`, it looked like there is a more widespread problem in the handling of the `VarOp` property. It expects one space on each side as the `oelint.vars.spacesassignment` rule calls for 😄 If there is more or less one space around to assignment operator the rules create invalid results.

E.g. A `bbclass` defines a default value for a variable. In a `bb` file (Notice the 2 spaces before `?=`) the variable is set to the actual value .. This leads to the `oelint_var_override` rule to be triggered for a perfectly valid use of the assignment operator.
```
# bbclass
NAME    ?= "default name"

# bb
inherit <bbclass>

NAME = "actual name"
```

For the `oelint.var.override` rule it can be easily fixed by the following approach (See #427)

```diff
                 # as these will be handled during parse time
                 # and apply to different rules
                 _items = [x for x in items if x.SubItem == sub and not x.IsAppend(
-                ) and x.VarOp not in [' := ', ' ?= ', ' ??= '] and not x.Flag]
+                ) and x.VarOp.strip() not in [':=', '?=', '??='] and not x.Flag]
                 if len(_items) > 1:
                     _files = {os.path.basename(x.Origin) for x in _items}
                     res += self.finding(_items[0].Origin, _items[0].InFileLine,
```

A quick scan of the rules showed at least 6 rules might be affected by this problem.

* oelint.append.protvars
* oelint.vars.appendop
* oelint.vars.bbvars
* oelint.vars.fileextrapathsop
* oelint.vars.srcuriappend
* oelint.var.override

**Comment by priv-kweihmann** [comment]
I see - I need to check if that needs to be fixed in the parser or here - but I get the problem and yeah that needs to be resolve either way
